front-end-node/search/SearchOverrides.js

```diff
@@ -0,0 +1,121 @@
+/*jshint browser:true, nonew:false*/
+/*global WebInspector:true, InspectorFrontendHost:true, InspectorFrontendHostAPI:true*/
+
+(function() {
+  var createSearchRegex = function(query, caseSensitive, isRegex)
+  {
+    var regexFlags = caseSensitive ? "g" : "gi";
+    var regexObject;
+
+    if (isRegex) {
+      try {
+        regexObject = new RegExp('^.*?'+query+'.*?$|^.*?'+query+'.*?\n|\n.*?'+query+'.*?\n|\n.*?'+query+'.*?$', regexFlags);
+      } catch (e) {
+        // Silent catch.
```

Comment:
is the try-catch block really necessary? which case the `RegExp` will throw an error?